test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -199,7 +203,8 @@ export default class BaseFormObject {
   }
 
   getAllErrorBoxTexts() {
-    return this.getInnerTexts(FORM_INFOBOX_ERROR);
+    const selector  = userVariables.v3 ? FORM_INFOBOX_ERROR_SELECTOR : FORM_INFOBOX_ERROR;
```

Comment:
For v3, is is possible to do this via `getByRole('alert')` and then sub select on the `.infobox-error` class?